config/exporters/monitoring/grafana/base/prometheus-exporter-service.yaml

```diff
@@ -66,9 +83,14 @@ spec:
       containers:
       - name: grafana-datasource-exporter
         image: quay.io/redhat-appstudio/o11y-prometheus-exporters:v0.1
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://127.0.0.1:8081/"
+        - "--logtostderr=true"
+        - "--v=10"
```

Comment:
Are those arguments that our image uses?

Reply:
removed